src/one_py_sdk/enterprise/report.py

```diff
@@ -0,0 +1,65 @@
+import requests
+import json
+import logging
+from one_py_sdk.enterprise.authentication import AuthenticationApi
+from one_py_sdk.shared.helpers.protobufhelper import DeserializeResponse
+
+class ReportApi:
+    def __init__(self, env, auth: AuthenticationApi):
+        self.AppUrl = "/enterprise/report/v1/"
+        self.Environment = env
+        self.Authentication = auth
+        
+    def GetReportDefinitions(self, plantId=None):
```

Comment:
do we want to use plantId or operationId?

Reply:
Everywhere else in the code it is plantId. I could go and change it everywhere, but for now I thought I'd just keep it consistent.

Reply:
I like consistency.  If the SDK consumer is already using PlantID, I'd keep it that way.  Thanks!